test/cpp/jit/test_base.cpp

```diff
@@ -0,0 +1,25 @@
+#include <test/cpp/jit/test_base.h>
+#include <test/cpp/jit/test_utils.h>
+
+#include "torch/csrc/jit/runtime/custom_operator.h"
+
+namespace torch {
+namespace jit {
+inline c10::OperatorOptions aliasAnalysisFromSchema() {
+  c10::OperatorOptions result;
+  result.setAliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA);
+  return result;
+}
+
+RegisterOperators reg({
+    Operator(
+        "prim::MakeTestTensor() -> Tensor",
+        [](Stack& stack) {
+          push(stack, at::Tensor());
```

Comment:
NB: if you ever try to actually run these tests these are going to produce undefined tensors, which will probably blow up in unexpected ways. So it's probably worth documenting MakeTestTensor to this effect.